tasks/util.py

```diff
@@ -0,0 +1,43 @@
+from os.path import dirname, realpath, expanduser, join, exists
+from shutil import rmtree
+from os import makedirs
+
```

Comment:
This file lets us share constants across tasks without duplicating in different scripts.